docs/integrations/sources/sftp.md

```diff
@@ -19,12 +19,12 @@ Key pair is created (typically by the user). This is typically done with ssh-key
 Private key stays with the user (and only there), while the public key is sent to the server. Typically with the ssh-copy-id utility.
 Server stores the public key (and "marks" it as authorized).
 Server will now allow access to anyone who can prove they have the corresponding private key.
-## Step 2: Set up the <connector name> connector in Airbyte
+## Step 2: Set up the `$CONNECTOR_NAME` connector in Airbyte
```

Comment:
JSX (a part of the docusuaurs language) uses HTML like tags.  It see `<literally anything>` in between `<` and `>` as valid code, so it sees `<connector name>` as an opening tag that is missing `</connector>` as a closing tag.  

this error shows up as `Expected corresponding JSX closing tag for </connector>` not found.

Reply:
@Amruta-Ranade this is the weirdness I was talking about 

@suhomud tagging you for a heads up on this oddity of docusuaurs, feel free to change my edits in a future PR.  thanks for your contributions <3